opentimestamps/calendar.py

```diff
@@ -59,7 +60,7 @@ def submit(self, digest, timeout=None):
 
         Returns a Timestamp committing to that digest
         """
-        req = urllib.request.Request(self.url + '/digest', data=digest, headers=self.request_headers)
+        req = urllib.request.Request(urljoin(self.url, '/digest'), data=digest, headers=self.request_headers)
```

Comment:
`urljoin()` doesn't need the slash at the beginning of `/digest` - it's meant to join URL fragments together, so the extra slash is redundant.